### DISCUSSION TITLE
[Esame 23 Febbraio 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#270] [Esami - Bertini @ Informatica]
**Esercizio 2.** Ogni giorno Tacito va a pescare alla valle della trota, tale valle ha due laghi, il lago d'oro ha solo trote, per il lago d'argento si stima che il 60% del totale dei pesci siano trote. Tacito si trova al bivio per i due laghi, non ricordando la strada, lancia una moneta equa, se esce testa va a destra, altrimenti a sinistra. Tacito sicuramente pescherà un pesce in quella giornata, ed una volta pescato un pesce, quale esso sia, torna a casa e termina la giornata. 

**(i)** Calcolare la probabilità che Tacito peschi una trota.

**(ii)** Sapendo che Tacito ha preso una trota. Calcolare la probabilità che l'abbia presa al lago d'oro.

**(iii)** Tacito il giorno seguente all'aver pescato una trota, ritorna allo stesso lago del giorno pretendete, seppur non sapendo quale sia, Calcolare la probabilità che peschi nuovamente una trota.

**Comment by Traster-uni** [comment]
[
![WhatsApp Image 2024-02-27 at 2 09 39 PM](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/84441861/0c9a9792-e3f1-4f35-8eaa-bbb1c2033f95)
](url)

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/e87c59fc-a3a1-4412-9f84-1e7150567d53)
